### DISCUSSION TITLE
Erro de execução de debug

### DISCUSSION THREAD

**Discussion by franciscoguerra** [balta-io/forum#23] [Tira Dúvidas] [answered]
![erro](https://user-images.githubusercontent.com/12381369/191889752-00ce4070-f8e6-41f0-bd9a-d664a1d18ad9.png)

não sei corrigir o erro.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @franciscoguerra, como vai? Qual curso, módulo e aula seriam?

**Reply by franciscoguerra** [reply]
Bom dia
 Carreira beckend .NET, Acesso a dados com .NET, C#, Dapper e Sql Serve.
Aula Dapper Contrib

creio que o erro é no login do banco, revisei varias coisas do curso más não consegui ainda resolver.

**Comment by franciscoguerra** [comment]
Boa Tarde, blz ?
 Carreira -> Desenvolvedor Beckend .NET
Curso -> Acesso a dados com .NET, C#, Dapper e SQL Serve.
aula->  Mão na massa -> Criando o projeto.



Em sex., 23 de set. de 2022 às 09:43, Andre Baltieri <
***@***.***> escreveu:

**Comment by franciscoguerra** [comment]
aula-> Dapper Contrib

Em sex., 23 de set. de 2022 às 13:07, Francisco Neto ***@***.***>
escreveu:

**Reply by andrebaltieri** [reply]
Bom dia, @franciscoguerra como vai?

No caso ele diz que sua senha está errada, você se lembra se colocou algo diferente? tem como testar outra senha?

Em adicional, temos um artigo que sempre seguimos nos cursos:
https://balta.io/blog/sql-server-docker

Ele mostra a instalação do SQL Server via Docker... Se estiver no Windows, não precisa do Docker, mas precisa da senha do SQL Server para se conectar de qualquer forma.
